R/map_fetch.R

```diff
@@ -0,0 +1,268 @@
+#' @title Fetch aggregated density maps of GBIF occurrences
+#'
+#' @export
+#'
+#' @description This function is a wrapper for the GBIF mapping api version 2.0.
+#' The mapping API is a web map tile service making it straightforward to 
+#' visualize GBIF content on interactive maps, and overlay content from other 
+#' sources. It returns tile maps or vector maps with number of 
+#' GBIF records per area unit that can be used in a variety of ways, for example
+#' in interactive leaflet web maps. Map details are specified by a number of 
+#' query parameters, some of them optional. Full documentation of the GBIF
+#' mapping api can be found at https://www.gbif.org/developer/maps
+#'
+#' @param source (character): Either density for fast, precalculated tiles, 
+#' or adhoc for any search. (\code{default = "density"})

```

Comment:
we're using markdown docs now, see https://cran.rstudio.com/web/packages/roxygen2/vignettes/markdown.html for help on that